app/src/main/res/drawable/tx_detail_header_shadow_gradient.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We have `scroll_height_gradient.xml`, can we reuse that one here? Maybe rename that to `view_elevation_gradient.xml` & reuse.